CHANGES

```diff
@@ -1,3 +1,7 @@
+0.174
+- Add configuration property to include and exclude HTTPS cipher suites for
```

Comment:
`Add configuration properties to ... for the HTTP client.` (note the period at the end).

I don't know how comprehensive we want to be here, but we can also tell the config property names. 

/cc @electrum